mass/ports/inbound/query_handler.py

```diff
@@ -38,9 +38,30 @@ def __init__(self):
         )
 
 
+class DeletionFailedError(RuntimeError):
+    """Raised when a deletion attempt fails because the ID can't be found"""
+
+    def __init__(self, resource_id: str):
+        super().__init__(
+            f"Failed to delete resource with ID '{resource_id}' because no match was "
+            + "found in the database."
+        )
+
+
 class QueryHandlerPort(ABC):
     """Port for the query handler"""
 
+    @abstractmethod
+    async def delete_resource(self, *, resource_id: str, class_name: str):
+        """Delete resource with given ID and class name from the database
+
+        Raises:
+            ClassNotConfiguredError - when the class_name parameter does not
+                match any configured class
+            FailedDeletionError - when the provided ID doesn't match any resource
```

Comment:
```suggestion
            DeletionFailedError - when the provided ID doesn't match any resource
```

Reply:
Changed in [31fbd56](https://github.com/ghga-de/mass/pull/10/commits/31fbd5612e1c31489bed7c951dd4db8ecf394f40)

Reply:
But the docstring here stil has the old name of the error.

Reply:
Oh man, yeah I was just thinking of the error name itself when I read the snippet above. Fixed it.